Index: firefox-22.0~b1+build1/js/src/jit-test/tests/parallelarray/bug858582.js
===================================================================
--- firefox-22.0~b1+build1.orig/js/src/jit-test/tests/parallelarray/bug858582.js	2013-05-15 11:19:17.000000000 +0100
+++ firefox-22.0~b1+build1/js/src/jit-test/tests/parallelarray/bug858582.js	2013-05-16 01:16:13.107403220 +0100
@@ -1,5 +1,6 @@
 // |jit-test| error: TypeError
 // Don't crash.
+if (getBuildConfiguration().parallelJS) {
 gczeal(2);
 evaluate("\
 function assertAlmostEq(v1, v2) {\
@@ -62,3 +63,6 @@
 			} )      
 			!= "x"; 
 	});
+} else {
+  throw new TypeError();
+}
