Description: Don't bundle our own hyphenation patterns
Author: Chris Coulson <chris.coulson@canonical.com>
Forwarded: not-needed
Bug-Ubuntu: https://launchpad.net/bugs/894166

Index: firefox-trunk-21.0~a1~hg20130213r121727/browser/installer/package-manifest.in
===================================================================
--- firefox-trunk-21.0~a1~hg20130213r121727.orig/browser/installer/package-manifest.in	2013-02-14 02:06:50.000000000 +0000
+++ firefox-trunk-21.0~a1~hg20130213r121727/browser/installer/package-manifest.in	2013-02-14 21:38:06.789965559 +0000
@@ -39,7 +39,7 @@
 @BINPATH@/browser/defaults/profile/localstore.rdf
 @BINPATH@/browser/defaults/profile/mimeTypes.rdf
 @BINPATH@/dictionaries/*
-@BINPATH@/hyphenation/*
+; @BINPATH@/hyphenation/*
 @BINPATH@/browser/@PREF_DIR@/firefox-l10n.js
 @BINPATH@/browser/searchplugins/*
 #ifdef XP_WIN32
